modules/system-configuration/src/main/java/org/apache/ignite/internal/configuration/SystemConfigurationSchema.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.configuration;
+
+import org.apache.ignite.configuration.annotation.ConfigurationRoot;
+import org.apache.ignite.configuration.annotation.ConfigurationType;
+import org.apache.ignite.configuration.annotation.Value;
+
+/**
+ * System Configuration schema.
+ */
+@ConfigurationRoot(rootName = "system", type = ConfigurationType.LOCAL)
+public class SystemConfigurationSchema {
```

Comment:
I'm not sure that this is a good naming. I would name it like 'PathsConfigurationSchema`. As I understood, this class will have other paths, not only `partitionsBasePath`. The same for root, I would name it `paths`

Reply:
I agree with Mirza on this, "system" sounds too vague in this context